vars/buildDockerAndPublishImage.groovy

```diff
@@ -6,7 +6,6 @@ import java.text.DateFormat
 def call(String imageName, Map userConfig=[:]) {
   def defaultConfig = [
     agentLabels: 'docker || linux-amd64-docker', // String expression for the labels the agent must match
-    builderImage: 'jenkinsciinfra/builder:2.0.2', // Version managed by updatecli
```

Comment:
Didn't found any corresponding manifest 🤷

Reply:
yep, we were stuck to this version because Alpine / IMG issues.